Router and basic structure

ark2002 opened the pull request:
- Added Router and links for routing
- Added Placeholders for Screens
- Did the Barrel exports as suggested in the previous PR

HarshnaNHaswani reviewed (approved):
Looks good to me

HarshithaKonduru214 commented on common.css:
```diff
@@ -0,0 +1,33 @@
+    margin: 2rem;
+    border: 0;
+    height: 4px;
+    background-image: linear-gradient(to right, #ffffff00, #ffffffbf, #ffffff00);
```
Can try using variables instead of hexcodes.

HarshithaKonduru214 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+  font-family: var(--primary-font-family);
+  background-color: var(--white-color);
+  align-items: center;
+  gap: 0;
```
This property can be removed as the value is anyway 0.

HarshithaKonduru214 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+  transform: translate(-50%, -50%);
+  font-family: var(--primary-font-family);
+  color: var(--white-color);
+  text-shadow: 0 0 7px #fff;
```
Can use variables for colors.

ark2002 replied:
done

HarshithaKonduru214 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+}
+
+.banner3__container>.card>.card--hover>.card__header {
+  background-color: var(--white-color);
+  color: var(--black-color);
```
Same as above, can make it variables

HarshithaKonduru214 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+  background-color: var(--black-color);
+  padding: 2rem 0rem;
+  text-align: center;
+  color: white;
```
same as above, make it into a variable.

HarshithaKonduru214 reviewed (approved):
Suggested a few minor changes, the rest looks good to me.

SRUJANA-PENUGONDA13 commented on index.html:
```diff
@@ -0,0 +1,194 @@
+  <div class="landingpage__container flex--column">
+    <header class="header flex--row">
+      <a href="">
+        <div class="header__logo-container flex--row">
```
Try to follow consistent names format

SRUJANA-PENUGONDA13 commented on index.html:
```diff
@@ -0,0 +1,194 @@
+          </div>
+        </div>
+      </div>
+      <button class="btn btn-color--primary btn__outline btn-font--secondary">
```
Try to follow consistent name formats

Ex - btn-font--secondary

SRUJANA-PENUGONDA13 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+.banner1__container {
+  height: 100%;
+  font-family: var(--primary-font-family);
+  background-color: var(--white-color);
```
It is not good approach to defining variables with colour names. If it is banner bg color define it with the same name So that in future if you want to change banner bg color it to some other color that name make sense but not yours.

SRUJANA-PENUGONDA13 commented on index.css:
```diff
@@ -0,0 +1,165 @@
+}
+
+.banner1__left>.btn {
+  color: var(--black-color);
```
same here

SRUJANA-PENUGONDA13 reviewed (commented):
Try to give generic names to colours instead of colour names itself. Everything looks good. Keep up good work

rahul5564 reviewed (commented):
cool

ark2002 committed:
feat : Added Router and basic structure

- Added Router and links for routing
- Added Placeholders for Screens
- Did the Barrel exports as suggested in the previous PR

ark2002 committed:
fix : deploy error

ark2002 committed:
fix : deploy error

ark2002 committed:
fix : deploy error

ark2002 committed:
fix:deploy error

MarkVed17 commented on src/frontend/components/Banners/index.js:
```diff
@@ -0,0 +1,3 @@
+export { QouteBanner } from "./QouteBanner/QouteBanner"
```
check for spelling err

MarkVed17 replied:
Quote*

ark2002 replied:
yup ill look for it

MarkVed17 reviewed (approved):
The structure looks proper Aryak.
Just check for spelling errors & code formatting.

akash-1318 commented on src/frontend/components/Banners/TitleBanner/TitleBanner.js:
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import "./TitleBanner.css";
+import { Link } from "react-router-dom"
```
You can also import Link in same way as you are importing Router from frontend file.

akash-1318 reviewed (commented):
Just added one suggestion look into that else everything looks good from my end.
